assets/src/editor/events/events/data/with-event-date-entities.js

```diff
@@ -8,35 +8,40 @@ import { createHigherOrderComponent } from '@wordpress/compose';
 const EMPTY_OBJECT = {};
 
 /**
- * withEventDatetimes
+ * withEventDateEntities
  * returns an object containing the following:
- *    eventDates - an array of datetime entities for the supplied event
- *    eventDatesLoaded - boolean true if loading is complete
+ *    dateEntities - an array of datetime entities for the supplied event
+ *    dateEntitiesLoaded - boolean true if loading is complete
  *
  * !!! IMPORTANT !!!
  * ONLY USE THIS AFTER THE CALL TO hasFinishedResolution() in the
  * withEvent() DATA HOC has completed and resolved true
  *
  * @function
  */
-export const withEventDatetimes = createHigherOrderComponent(
+const withEventDateEntities = createHigherOrderComponent(
```

Comment:
No, this was intentional.  It retains the clarity that this will return `dateEntities` for a given event.  See [this comment](https://github.com/eventespresso/event-espresso-core/pull/1295/files#r292448190) for more rationale behind this pattern.